pkg/backend/url.go

```diff
@@ -39,8 +43,16 @@ func FromPath(path string) *URL {
 	}
 }
 
+var winPath = regexp.MustCompile(`^(?:[a-zA-Z]\:|\\\\[\w\.]+\\[\w.$]+)\\(?:[\w]+\\)*\w[\w.\. \\-]+$`)
+
 // ParseURL attempts to parse an backend URL
 func ParseURL(us string) (*URL, error) {
+	// url.Parse does not handle windows paths very well
+	// see: https://github.com/golang/go/issues/13276
+	if runtime.GOOS == "windows" && winPath.MatchString(us) {
+		us = fmt.Sprintf("file:///%s", us)
```

Comment:
This is a hacky way to support windows paths here. It works, but it only supports the drive letter schema `C:\Users\johndoe`, not the UNC file paths `\\hostname\dir\file`.

Reply:
OK, not an issue for now. I think its `\\hostname\share\path_in_share` right? So we could match that via regexp as well, no?